src/installation/guides/fde.md

```diff
@@ -270,10 +275,24 @@ Ensure an initramfs is generated:
 # xbps-reconfigure -fa
 ```
 
-Exit the `chroot`, unmount the filesystems, and reboot the system.
+Exit the `chroot`, unmount the filesystems. 
 
 ```
 # exit
 # umount -R /mnt
+```
+
+If you are using a LVM configuration you will need to run the following commands before continuing with the rest of the commands.
+
+```
+# lvchange -an voidvm
+
+# cryptsetup close /dev/voidvm/root
+```
+
+Finally, Close the encrypted partition and reboot the system.
+
+```
+# cryptsetup close /dev/mapper/voidvm
```

Comment:
Does any of this matter? If so:

```suggestion
# lvchange -an voidvm
# cryptsetup close /dev/mapper/voidvm
```

Otherwise, just drop all of this and go straight to the reboot.